cloudevents/translated/zh-cn/spec_CN.md

```diff
@@ -6,7 +6,7 @@ CloudEvents 是一个用于定义事件格式的供应商中立规范。
 
 ## 文档状态
 
-这份文档是一份仍在制作中的工作草稿。
+这份文档是一份仍在制作中的工作草案。
```

Comment:
what changed here? and in the previous edit

Reply:
> previous last two words "草稿" is translated from "draft". But it's not quite formal here. So I use "草案" to replace it.

Reply:
ah now I see it - thanks!